### PR TITLE
Display and sync pointers between clients

### PR DESCRIPTION
**Pull request by martinRenou**
This PR:
- [x] Removes the color-picking logic and implements a BVH picking instead, this has the benefit of being allowed to **retrieve the projected pointer position when picking**
- [x] Displays all the user pointers, using the user color
- [x] Removes the "highlight on hover" feature, as it does not give any more visual information than the pointer now
- [x] Re-implements the camera syncing between users, syncing the entire camera position and orientation information (position, rotation, up, target). This will be useful later when implementing a "follow" feature

Demo: (the user on the right side is hovering the meshes with his mouse, and his pointer is visible to any contributor)

[Screencast from 11-09-2022 04:21:43 PM.webm](https://user-images.githubusercontent.com/21197331/200871010-b2686f6e-d6c9-4be3-b24a-46daa0fcce82.webm)

### CODE REVIEW

File: src/mainview.tsx [hunk 1]
```diff
@@ -382,20 +357,39 @@ export class MainView extends React.Component<IProps, IStates> {
   private _onPointerMove(e: MouseEvent) {
     const rect = this._renderer.domElement.getBoundingClientRect();
 
-    this._pointer.x = e.clientX - rect.left;
-    this._pointer.y = e.clientY - rect.top;
+    this._pointer.x = ((e.clientX - rect.left) / rect.width) * 2 - 1;
+    this._pointer.y = -((e.clientY - rect.top) / rect.height) * 2 + 1;
```
Comment (trungleduc) [inline src/mainview.tsx:361]: I'm just curious, what does this formula mean?

**Comment by trungleduc** [comment]
> * [ ]  Re-implements the camera syncing between users, syncing the entire camera position and orientation information (position, rotation, up, target). This will be useful later when implementing a "follow" feature
> 

This task is done?

File: src/mainview.tsx (hunk 1 above)
Comment (martinRenou) [reply src/mainview.tsx:361]: It scales the DOM coordinates into viewport coordinates (in the range `[-1, 1]`)

Comment (trungleduc) [reply src/mainview.tsx:361]: Thanks!

**Comment by trungleduc** [comment]
Thanks @martinRenou !

**Review by trungleduc — APPROVED** [review]

**Pull request merged**